cmd/spec/example_reservation.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/RHEnVision/provisioning-backend/internal/ptr"
 )
 
-var ReservationTime time.Time = MustParseTime("2013-05-13T19:20:25Z")
+var ReservationTime = MustParseTime("2013-05-13T19:20:25Z")
```

Comment:
There are couple of changes like this, I did not pay attention and IDE fixed these for me. These are not relevant, will not hurt I guess.

Reply:
I like it, that's the way C++ headed last time I lookes as well, no useless explicity :)